content/en/standard-attributes/_index.md

```diff
@@ -0,0 +1,1807 @@
+      - icon-rum
+      - browser
+    type: string
+    domain: Source errors (Browser events)
+
+  - name: action.frustration.type:dead_click
+    description: The dead clicks detected by the RUM Browser SDK.
+    product_source:
+      - icon-rum
+      - browser
+    type: string
+    domain: Frustration signals (Browser events)
+  
+  - name: action.frustration.type:rage_click
+    description: The rage clicks detected by the RUM Browser SDK.
+    product_source:
+      - icon-rum
+      - browser
+    type: string
+    domain: Frustration signals (Browser events)
+  
+  - name: action.frustration.type:error_click
+    description: The error clicks detected by the RUM Browser SDK.
+    product_source:
+      - icon-rum
+      - browser
+    type: string
+    domain: Frustration signals (Browser events)
+  
+  - name: view.url_query.utm_source
+    description: The parameter in the URL tracking the source of traffic.
+    product_source:
+      - icon-rum
+      - browser
+    type: string
+    domain: UTM (Browser events)
+  
+  - name: view.url_query.utm_medium
+    description: The parameter in the URL tracking the channel where the traffic is coming from.
+    product_source:
+      - icon-rum
+      - browser
+    type: string
+    domain: UTM (Browser events)
+  
+  - name: view.url_query.utm_campaign
+    description: The parameter in the URL identifying the specific marketing campaign tied to that view.
+    product_source:
+      - icon-rum
+      - browser
+    type: string
+    domain: UTM (Browser events)
+  
+  - name: view.url_query.utm_content
+    description: The parameter in the URL identifying the specific element a user clicked within a marketing campaign.
+    product_source:
+      - icon-rum
+      - browser
+    type: string
+    domain: UTM (Browser events)
+  
+  - name: view.url_query.utm_term
+    description: The parameter in the URL tracking the keyword a user searched to trigger a given campaign.
+    product_source:
+      - icon-rum
+      - browser
+    type: string
+    domain: UTM (Browser events)
+  
+
+
+  - name: language
+    description: The client SDK language used to generate the span. It can be one of cpp, dotnet, go, jvm, javascript, php, python, ruby.
+    product_source:
+      - icon-apm
+    type: string
+    domain: APM core
+  - name: env
+    description: The value of `DD_ENV` environment variable or user defined `env` for the running process.
+    product_source:
+      - icon-apm
+    type: string
+    domain: APM core (Reserved)
+  - name: version
+    description: The value of `DD_VERSION` environment variable or user defined `version` for the running process.
+    product_source:
+      - icon-apm
+    type: string
+    domain: APM core (Reserved)
+  - name: span.kind
+    description: The string representing the type of work unit handled by the span. It can be   one of server, client, producer, consumer or internal. More information in the   OpenTelemetry SpanKind documentation.
```

Comment:
```suggestion
    description: The string representing the type of work unit handled by the span. It can be one of server, client, producer, consumer, or internal. For more information, see the [OpenTelemetry SpanKind documentation](https://opentelemetry.io/docs/specs/otel/trace/api/#spankind).
```